doc/source/reference/api/eventlet.hubs.rst

```diff
@@ -44,6 +44,14 @@ eventlet.hubs.poll module
    :undoc-members:
    :show-inheritance:
 
+eventlet.hubs.pyevent module
```

Comment:
Did you mean to restore this?

Reply:
Oh good catch. I think I forgot to rebase my development branch... Will fix that